docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -0,0 +1,77 @@
+//[id="proc_importing_code_{context}"]
+
+= Importing code to App Studio
```

Comment:
```suggestion
= Importing code to {ProductName}
```
Should this be abstracted?

Reply:
logically, @mattreid, it should, but our publishing tool chokes on var tas in titles/headings. i changed the whole title anyway per a request from Tanya.